app/views/api/integrations/apicast/configuration_driven/_apicast.html.slim

```diff
@@ -50,13 +51,20 @@ section.Configuration class=('Configuration--is-promoted' unless deployment_opti
       article.Environment-summary data-state="open" class="Environment--#{@show_presenter.test_state_modifier}"
         h3.Environment-title Staging Environment
         - if @show_presenter.any_sandbox_configs?
-          = @show_presenter.staging_proxy_endpoint
-          .ConfigurationVersion
-            ' v.
-            = @show_presenter.last_sandbox_config.version
-          = semantic_form_for @proxy, url: promote_to_production_admin_service_integration_path(@service, anchor: 'production'), html: { class: 'PromoteForm' } do |f|
-            = f.buttons class: "buttons buttons-inline" do
-              = f.button *promote_to_production_button_options(@show_presenter)
+          dl.u-dl
+            dt.u-dl-term URL
+            dd.u-dl-definition = @show_presenter.staging_proxy_endpoint
+            dt.u-dl-term Version
+            dd.u-dl-definition
+              ' v.
+              = @show_presenter.last_sandbox_config.version
+            dt.u-dl-term Test Curl
+            dd.u-dl-definition
+              = api_test_curl(@proxy)
```

Comment:
A couple enhancements to do here:
1. Probably render only if the product has applications, similar to https://github.com/3scale/porta/blob/22b1411b8eda4b4d1e54818d89db209f39c9dbf6/app/views/api/integrations/apicast/shared/_curl.html.slim#L4
2. Add the path for the first backend to the URI – requires refactoring of https://github.com/3scale/porta/blob/22b1411b8eda4b4d1e54818d89db209f39c9dbf6/app/helpers/api/integrations_helper.rb#L3